### PR TITLE
Add Memory Request for Telegraf

### PR DESCRIPTION
**Pull request by polinaLyubavina**
To fix the memory allocation issue with Telegraf, I added a request for memory in `values.yaml` and also added `resources.requests.memory` to `deployment.yaml` so the users can modify the config file for their memory usage. Hopefully this fix will also fix another error that was emerging. 

This is the issue I had:
 `Warning  FailedPostStartHook  36m (x2 over 36m)    kubelet            Exec lifecycle hook ([/bin/bash /etc/setup/setup.sh]) for Container "telegraf" in Pod "grnoc-telegraf-f55756c8f-96t8d_slate-group-catalog-test-prod(a866e656-0420-40bf-aa30-ae3122665b79)" failed - error: command '/bin/bash /etc/setup/setup.sh' exited with 137: , message: ""`

The second error was:
`  Warning  FailedPostStartHook  36m                  kubelet            Exec lifecycle hook ([/bin/bash /etc/setup/setup.sh]) for Container "telegraf" in Pod "grnoc-telegraf-f55756c8f-96t8d_slate-group-catalog-test-prod(a866e656-0420-40bf-aa30-ae3122665b79)" failed - error: rpc error: code = Unknown desc = failed to exec in container: failed to start exec "ae3b437f9250e690458d07ad2f263e90c82e48b194cb680b7536bacc98866b8f": OCI runtime exec failed: exec failed: unable to start container process: error executing setns process: exit status 1: unknown, message: ""`

### CODE REVIEW

File: charts/grnoc-telegraf/templates/deployment.yaml [hunk 1]
```diff
@@ -43,6 +43,9 @@ spec:
                 secretKeyRef:
                   name: {{ .Values.grnocOutput.passwordSecretName }}
                   key: password
+          resources:
+            requests: 
+              memory: '{{.Values.requestMemory}}'
```
Comment (sthapa) [inline charts/grnoc-telegraf/templates/deployment.yaml:48]: Just a nit, but can you add spaces before and after  the braces, e.g. like the entry above.  I don't think it'll cause template errors but it looks better and might prevent an error

File: charts/grnoc-telegraf/values.yaml [hunk 2]
```diff
@@ -9,6 +9,9 @@ Instance: ""
 # Optionally write metrics to stdout and local file in container.
 writeToStdout: true
 
+# Specifies the amount of memory (in GB) to request
+requestMemory: "1000"
```
Comment (sthapa) [inline charts/grnoc-telegraf/values.yaml:13]: Looks like this value is requesting 1000GB according to the comment above, is that accurate?

**Review by sthapa — COMMENTED** [review]
Could you look at the two items I mentioned and change if needed?

**Review by sthapa — APPROVED** [review]
LGTM

File: charts/grnoc-telegraf/values.yaml (hunk 2 above)
Comment (polinaLyubavina) [reply charts/grnoc-telegraf/values.yaml]: ooopsies, changed that to "1G"

File: charts/grnoc-telegraf/templates/deployment.yaml (hunk 1 above)
Comment (polinaLyubavina) [reply charts/grnoc-telegraf/templates/deployment.yaml]: added the spaces; will keep this in mind for the future

**Pull request merged**